staging/src/k8s.io/kubectl/pkg/validation/schema_test.go

```diff
@@ -139,3 +142,97 @@ func TestConjunctiveSchema(t *testing.T) {
 		})
 	}
 }
+
+type mockVerifier struct {
+	supported bool
+}
+
+func (v *mockVerifier) HasSupport(gvk schema.GroupVersionKind) error {
+	if !v.supported {
+		return resource.NewParamUnsupportedError(gvk, resource.QueryParamFieldValidation)
```

Comment:
I think, there is no need to export that `NewParamUnsupportedError` function just because of this test. As far as I can see, you are checking error is nil or not. That would be better to return standard error for mocking?